packages/adblocker/src/resources.ts

```diff
@@ -21,13 +23,44 @@ function btoaPolyfill(buffer: string): string {
 }
 
 export interface Resource {
+  names: string[];
+  body: string;
   contentType: string;
+}
+
+interface Scriptlet {
+  names: string[];
   body: string;
+  dependencies: string[];
+  executionWorld: Scripting.ExecutionWorld;
+  requiresTrust: boolean;
+}
+
+export interface ResourcesDistribution {
+  redirects: Array<{
+    names: string[];
+    content: string;
+    contentType: string;
+    encoding?: 'base64';
+  }>;
+  scriptlets: Array<{
+    names: string[];
+    content: string;
+    dependencies: string[];
+    executionWorld?: Scripting.ExecutionWorld;
+    requiresTrust?: boolean;
+  }>;
 }
 
-// TODO - support # alias
 // TODO - support empty resource body
 
+export const wrapScriptletBody = (script: string, dependencies: string[]): string =>
```

Comment:
lets add a default value for dependencies so it can be called without the second argument in tests